### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by DennisAgyekum**

### CODE REVIEW

**Comment by DennisAgyekum** [comment]
i mistakenly gave the wrong message to my Ez Namey work and i do not know how to change it.

File: javaScript-week1/Ez-Namey.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "Romeo",
+    "Sierra",
+    "Tango",
+];
+
+const randomNumber1 = Math.floor(Math.random() * 10); 
+const randomNumber2 = Math.floor(Math.random() * 10); 
+const startupName = firstWords[randomNumber1] + " " + secondWords[randomNumber2];
+console.log(
+  `"The startup: ${startupName}, contains ${startupName.length} characters`
```
Comment (jason-vasilev) [inline javaScript-week1/Ez-Namey.js]: There is a quote mark you can remove right before `The`

File: javaScript-week1/Ez-Namey.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+let firstWords = [
```
Comment (jason-vasilev) [inline javaScript-week1/Ez-Namey.js]: Use `const` until the browser doesn't `let` you to.

File: javaScript-week1/Age-ify.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// Age-ify
+
+const yearOfBirth = 1989;
+let yearFuture = 2028;
```
Comment (jason-vasilev) [inline javaScript-week1/Age-ify.js]: This could also be `const`

File: javaScript-week1/Goodboy-Oldboy.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+// Goodboy-Oldboy
+
+const dogYearOfBirth = 2020;
+let dogYearFuture = 2030;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = false;
+
+if (shouldShowResultInDogYears === false) {
```
Comment (jason-vasilev) [inline javaScript-week1/Goodboy-Oldboy.js]: The logic is inverted here. You check if `shouldShowResultInDogYears` is not true, which would mean to show the age in human years, but you do multiply it by 7, and show dog years.
Also `shouldShowResultInDogYears === false` can be written as `!shouldShowResultInDogYears` for shorter.

File: javaScript-week1/Goodboy-Oldboy.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+// Goodboy-Oldboy
+
+const dogYearOfBirth = 2020;
+let dogYearFuture = 2030;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = false;
```
Comment (jason-vasilev) [inline javaScript-week1/Goodboy-Oldboy.js]: Use `const` unless you are to reassign the values.

File: javaScript-week1/Housey-pricey.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
+// Housey pricey
+
+function sumVolume(width, height, depth) {
+    return width * height * depth;
+  }
+  
+  function trueHousePrice(volumeInMeters, gardenSize) {
+    return volumeInMeters * 2.5 * 1000 + gardenSize * 300;
+  }
```
Comment (jason-vasilev) [inline javaScript-week1/Housey-pricey.js:9]: Great job at using functions and making two, each for calculating a specific thing.

File: javaScript-week1/Housey-pricey.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    if (truePrice > costPrice) {
+      console.log(`paying too little. The real price is: ${truePrice}`);
+    } else {
+      console.log(
+        `paying more than the estimated cost. The real price is: ${truePrice}`
+      );
+    }
+  }
+  const marketPricePeter = 2.5e6;
+  const marketPriceJulia = 1e6;
```
Comment (jason-vasilev) [inline javaScript-week1/Housey-pricey.js:21]: I see this way of writing numbers for the first time. I see it works, but it's not the clearest and easiest. Could you share some article or blog post explaining it?

**Review by jason-vasilev — COMMENTED** [review]
I've left you a few suggestions, primary remembering to use `const` unless you are reassigning the values of the variables. Goodboy-Oldboy is not working, as the conditional logic is inverted. After you fix it in a new commit, I'll approve.

File: javaScript-week1/Ez-Namey.js (hunk 1 above)
Comment (DennisAgyekum) [reply javaScript-week1/Ez-Namey.js]: i am on it

File: javaScript-week1/Housey-pricey.js (hunk 7 above)
Comment (DennisAgyekum) [reply javaScript-week1/Housey-pricey.js:21]: https://www.youtube.com/watch?v=0ELC8R69Rkw&list=PL7TLF4T4Tq2TtNmadzRfxYIB9683uhpbD&index=10 
this is a link to the video.

**Review by jason-vasilev — APPROVED** [review]

**Pull request merged**